data.md

```diff
@@ -350,19 +350,21 @@ One needs to unname the `ValTypes` first before calling the `#take` or `#drop` f
 
 ```k
     syntax ValStack ::= #zero ( ValTypes )            [function]
-                      | #take ( Int , ValStack )      [function]
-                      | #drop ( Int , ValStack )      [function]
-                      | #revs ( ValStack )            [function]
-                      | #revs ( ValStack , ValStack ) [function, klabel(#revsAux)]
- // ------------------------------------------------------------------------------
+                      | #take ( Int , ValStack )      [function, functional]
+                      | #drop ( Int , ValStack )      [function, functional]
+                      | #revs ( ValStack )            [function, functional]
+                      | #revs ( ValStack , ValStack ) [function, functional, klabel(#revsAux)]
+ // ------------------------------------------------------------------------------------------
     rule #zero(.ValTypes)             => .ValStack
     rule #zero(ITYPE:IValType VTYPES) => < ITYPE > 0 : #zero(VTYPES)
 
-    rule #take(N, _)      => .ValStack               requires notBool N >Int 0
-    rule #take(N, V : VS) => V : #take(N -Int 1, VS) requires         N >Int 0
+    rule #take(N, _)         => .ValStack               requires notBool N >Int 0
```

Comment:
Can we at least add some documentation saying this case should never happen, and the same for `#drop`?

Reply:
I added a note saying that they have weird behavior on the empty valstack, and it's up to each call-site to determine if that's appropriate.